packages/ui/src/Providers/AppBoundary/AppMeta.tsx

```diff
@@ -0,0 +1,49 @@
+import { conditionalString } from '@silverhand/essentials';
+import classNames from 'classnames';
+import i18next from 'i18next';
+import { useContext } from 'react';
+import { Helmet } from 'react-helmet';
+
+import PageContext from '@/Providers/PageContextProvider/PageContext';
+import defaultAppleTouchLogo from '@/assets/apple-touch-icon.png';
+import defaultFavicon from '@/assets/favicon.png';
+
+import * as styles from './index.module.scss';
+
+/**
+ * User React Helmet to manage html and body attributes
+ * @see https://github.com/nfl/react-helmet
+ *
+ * - lang: set html lang attribute
+ * - data-color-mode: set html data-color-mode attribute
```

Comment:
cloud not find reference for this, do you have a reference link handy?

Reply:
Not really, simply provide a new way of color theme ref for those who want to set theme-based custom CSS.  Checked GitHub is using these attributes for their theme-based styles.